src/components/suggest/suggest_item.tsx

```diff
@@ -123,14 +150,20 @@ export const EuiSuggestItem: FunctionComponent<EuiSuggestItemProps> = ({
 
   let colorClass = '';
 
-  const labelDisplayClass = classNames(
+  const labelClassNames = classNames(
     'euiSuggestItem__label',
     labelDisplayToClassMap[labelDisplay],
     {
       'euiSuggestItem__labelDisplay--expand': !description,
```

Comment:
I'm noticing that this class name could actually get applied twice. If they both didn't supply a description and manually set the display to `expand`. Instead, I'd remove this line
```suggestion
    labelDisplayToClassMap[labelDisplay],
    {
```

And then...